src/ifElse.test.js

```diff
@@ -1,11 +1,57 @@
 'use strict';
 
 describe('ifElse', () => {
-  // const { ifElse } = require('./ifElse');
+  const { ifElse } = require('./ifElse');
 
-  it('should ', () => {
+  it('should call the second callback when condition returns false', () => {
+    let conditionCalled = false;
+    let firstCallbackCalled = false;
+    let secondCallbackCalled = false;
 
+    const condition = () => {
+      conditionCalled = true;
+
+      return false;
+    };
+
+    const firstCallback = () => {
+      firstCallbackCalled = true;
+    };
+
+    const secondCallback = () => {
+      secondCallbackCalled = true;
+    };
+
+    ifElse(condition, firstCallback, secondCallback);
+
+    expect(conditionCalled).toBeTruthy();
+    expect(firstCallbackCalled).toBeFalsy();
+    expect(secondCallbackCalled).toBeTruthy();
   });
 
-  // write tests here
```

Comment:
The comment 'write tests here' is not needed anymore, as tests are written. Please remove the comment.